### ISSUE TITLE
Funnel has service-info under /tasks/service-info, should be /service-info

### ISSUE THREAD

**Issue by mvdbeek** [ohsu-comp-bio/funnel#666]
Hi there, any chance you could move or mirror the /tasks/service-info to the standard /service-info route ?. That would make writing client implementations just a little easier. Thanks!

**Comment by kellrott** [comment]
Yes. Right now, we're trying to figure out the work of moving Funnel to TES 1.1. That should move Funnel to be better synchronized with the OpenAPI spec. 
In the mean time, we'll investigate a minor fix to address this issue.
